### PR TITLE
upgrading spec-version-maven-plugin

### PR DESCRIPTION
**Pull request by ankathur**

### CODE REVIEW

**Comment by ankathur** [comment]
@glassfishrobot run ci tests please

**Comment by ankathur** [comment]
@glassfishrobot run ci tests please

**Comment by ankathur** [comment]
@glassfishrobot run ci tests please

File: appserver/pom.xml [hunk 1]
```diff
@@ -136,7 +136,6 @@
     </properties>
 
     <modules>
-
```
Comment (arindam-bandyopadhyay) [inline appserver/pom.xml]: Can you please remove the new line.

**Review by arindam-bandyopadhyay — APPROVED** [review]
Looks Good

**Pull request merged**